src/config/index.ts

```diff
@@ -327,20 +327,20 @@ export const PUBLIC_REGISTER_NAME = "#public_register_name";
 export const ROLE_AND_RESPONSIBILITIES = "#role_and_responsibilities";
 export const CONTACT_FULL_NAME = "#contact_full_name";
 export const CONTACT_EMAIL = "#contact_email";
-export const CREATED_DATE = "#createdDateDay";
+export const CREATED_DATE = "#createdDate-createdDateDay";
 export const INCORPORATION_COUNTRY = "#incorporation_country";
 export const ENTITY_NAME = "#entity_name";
 export const ROLE_WITHIN_TRUST = "#roleWithinTrust";
 export const FORENAME = "#forename";
 export const SURNAME = "#surname";
-export const TRUST_DATE_OF_BIRTH = "#dateOfBirthDay";
-export const INTERESTED_PERSON_START_DATE = "#dateBecameIPDay";
+export const TRUST_DATE_OF_BIRTH = "#dateOfBirth-dateOfBirthDay";
+export const INTERESTED_PERSON_START_DATE = "#dateBecameIP-dateBecameIPDay";
 export const LEGAL_INTERESTED_PERSON_START_DATE = "#interestedPersonStartDateDay";
 export const PUBLIC_REGISTER_JURISDICTION = "#public_register_jurisdiction";
 export const LEGAL_ENTITY_NAME = "#legalEntityName";
 export const TRUST_BENEFICIAL_OWNERS = "#trustBeneficialOwners";
-export const TRUSTEE_START_DATE = "#startDateDay";
-export const TRUSTEE_END_DATE = "#endDateDay";
+export const TRUSTEE_START_DATE = "#startDate-startDateDay";
+export const TRUSTEE_END_DATE = "#endDate-endDateDay";
 export const CORPORATE_NAME = "#corporate_name";
 export const INDIVIDUAL_NAME = "#firstName";
 export const AGENT_CODE = "#agent_code";
```

Comment:
whats the purpose of these changes?

Reply:
Not relevant. Must have overlooked them. Will remove

Reply:
It's still showing some difference - if you don't mean/intend to change the values of these at all just checkout the file from main

git checkout main -- src/config/index.ts

Reply:
Still displaying changes here, can you please remove them :)

Reply:
Ah missed those. Done there now